src/test/java/org/rulez/demokracia/pdengine/choice/ChoiceDeleteAdminKeyIsUserTest.java

```diff
@@ -9,12 +10,17 @@
 import org.rulez.demokracia.pdengine.annotations.TestedFeature;
 import org.rulez.demokracia.pdengine.annotations.TestedOperation;
 import org.rulez.demokracia.pdengine.dataobjects.VoteAdminInfo;
+import org.rulez.demokracia.pdengine.exception.ReportedException;
 
 @TestedFeature("Manage votes")
 @TestedOperation("delete choice")
 @RunWith(MockitoJUnitRunner.class)
 public class ChoiceDeleteAdminKeyIsUserTest extends ChoiceTestBase {
 
+  private static final String NOT_SAME_USER_MESSAGE =
+      "The adminKey is \"user\" but the user is not same with that user who added the choice.";
+  private static final String CAN_ADDIN_IS_FALSE_MESSAGE =
```

Comment:
Aren't those strings are constants in the production code?

Reply:
They are. If you mean, we should use constants of the production code, I suggest to forget it.
Let's imagine the situation, when someone accidentally removes some words from this string or an unlucky regex overwrites it. We wouldn't realize it, since we compare this constant to itself.